Packs/SpurContextAPI/Integrations/SpurContextAPI/SpurContextAPI.yml

```diff
@@ -3,12 +3,22 @@ commonfields:
   id: SpurContextAPI
   version: -1
 configuration:
-- display: ""
+- display: "Server URL (e.g. https://api.spur.us/)"
+  name: base_url
+  type: 0
+  required: true
+  defaultvalue: https://api.spur.us/
+- display: ''
```

Comment:
no need to have display key here, you can remove it and keep the param like this:
- displaypassword: API token
  hiddenusername: true
  name: credentials
  required: true
  type: 9

Reply:
Removed display.